pkg/env/env.go

```diff
@@ -93,7 +93,17 @@ func setConfigLocation() (err error) {
 }
 
 func surveyMissingValues() error {
-	err := surveyConfigSource()
+	err := surveyManifests()
+	if err != nil {
+		return err
+	}
+
```

Comment:
Hey, we agreed to write such statement as:
``
if err := surveyManifests(); err != nil {
    return err
}
``
a while back. Could you change these before merge please.
See https://github.com/fusor/cpma/blob/d28c4c1fd45f70af783a20488ed8865c1169b27a/pkg/env/env.go#L46 for example.

Reply:
Yes you're right, this is actually legacy code we need to change. Thanks